github.js

```diff
@@ -16,6 +17,21 @@ var semver = require('semver');
 
 var which = require('which');
 
+function extend(dest) {
+  for (var i = 1, len = arguments.length; i < len; i++) {
+    var src = arguments[i];
+    for (var key in src) {
+      if (typeof src[key] === 'object') {
+        dest[key] = extend({}, src[key])
+      } else {
+        dest[key] = src[key];
+      }
+    }
+  }
+
+  return dest;
+}
```

Comment:
It was in case we wanted to put the configure under a shared key. I hate having to revisit code because I didn't put enough effort in the first time, but I guess I'm just handing this over so it can be someone else's issue.